crates/typed-store/src/rocks/iter.rs

```diff
@@ -58,10 +66,12 @@ impl<'a, K: DeserializeOwned, V: DeserializeOwned> Iterator for Iter<'a, K, V> {
                 .db_iter
                 .value()
                 .expect("Valid iterator failed to get value");
-            self.db_metrics
-                .rocksdb_iter_bytes
-                .with_label_values(&[&self.cf])
-                .observe((raw_key.len() + raw_value.len()) as f64);
+            if let Some(_timer) = report_metrics {
+                self.db_metrics
+                    .rocksdb_iter_bytes
+                    .with_label_values(&[&self.cf])
```

Comment:
I don't think the iter_bytes really needs to be sampled.  This is not as expensive as measuring latency.

Reply:
This one could actually be expensive as it is happening in the next() call which might just be iterating over data in memory.